### PR TITLE
[APP-2975] Design review v2

### PR DESCRIPTION
**Pull request by pedroagostinhoaptoide**
**What does this PR do?**

   - Fixes design issues in the second design review of AG. Specifically, it fixes typography issues and issues in the paddings and flexibility of the bonus views.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] See by commit.

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [APP-2975](https://aptoide.atlassian.net/browse/APP-2975)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2975](https://aptoide.atlassian.net/browse/APP-2975)

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

[APP-2975]: https://aptoide.atlassian.net/browse/APP-2975?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2975]: https://aptoide.atlassian.net/browse/APP-2975?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/MoreBonusViewFooter.kt [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+private fun MoreBonusViewPreview() {
+  Image(
+    imageVector = getMoreBonusViewFooter(Color.Green, Color.Black, Color.Magenta, Color.Gray),
+    contentDescription = null,
+    modifier = Modifier.size(240.dp)
+  )
+}
+
+fun getMoreBonusViewFooter(
+  backgroundColor: Color = Palette.Secondary,
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/MoreBonusViewFooter.kt:31]: Let's follow the convention we have about colors by naming them as `color1`, `color2` etc.
Reference to `getPaymentsProgressBackground` as an example.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/MoreBonusViewHeader.kt [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+private fun MoreBonusViewHeaderPreview() {
+  Image(
+    imageVector = getMoreBonusViewHeader(Color.Green, Color.Magenta, Color.Gray),
+    contentDescription = null,
+    modifier = Modifier.size(240.dp)
+  )
+}
+
+fun getMoreBonusViewHeader(
+  backgroundColor: Color = Palette.Secondary,
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/MoreBonusViewHeader.kt:31]: Let's follow the convention we have about colors by naming them as `color1`, `color2` etc.
Reference to `getPaymentsProgressBackground` as an example.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/WantMoreViewFooter.kt [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+      giftColor = Color.Magenta,
+      giftOutlineColor = Color.Gray
+    ),
+    contentDescription = null,
+    modifier = Modifier.size(240.dp)
+  )
+}
+
+fun getWantMoreViewFooter(
+  backgroundColor: Color = Palette.Secondary,
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/WantMoreViewFooter.kt:36]: Let's follow the convention we have about colors by naming them as `color1`, `color2` etc.
Reference to `getPaymentsProgressBackground` as an example.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/WantMoreViewHeader.kt [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+private fun WantMoreViewHeaderPreview() {
+  Image(
+    imageVector = getWantMoreViewHeader(Color.Green, Color.Black, Color.Magenta, Color.Gray),
+    contentDescription = null,
+    modifier = Modifier.size(240.dp)
+  )
+}
+
+fun getWantMoreViewHeader(
+  backgroundColor: Color = Palette.Secondary,
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/WantMoreViewHeader.kt:31]: Let's follow the convention we have about colors by naming them as `color1`, `color2` etc.
Reference to `getPaymentsProgressBackground` as an example.

File: app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/BonusSectionView.kt [hunk 5]
```diff
@@ -119,12 +119,16 @@ private fun BonusBannerText(
...
+    modifier = Modifier
+      .padding(start = 16.dp, top = 44.dp)
+      .width(240.dp)
   ) {
     AptoideOutlinedText(
-      text = stringResource(id = R.string.bonus_banner_title, "20"), //TODO Hardcoded value (should come from backend in the future)
+      text = stringResource(
+        id = R.string.bonus_banner_title,
+        "20"
+      ), //TODO Hardcoded value (should come from backend in the future)
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/BonusSectionView.kt:131]: Move the comment on the previous line next to `"20"`

**Review by G0retZ — CHANGES_REQUESTED** [review]

File: app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/BonusSectionView.kt (hunk 5 above)
Comment (pedroagostinhoaptoide) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/BonusSectionView.kt]: Fixed

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/WantMoreViewHeader.kt (hunk 4 above)
Comment (pedroagostinhoaptoide) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/WantMoreViewHeader.kt]: Fixed

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/WantMoreViewFooter.kt (hunk 3 above)
Comment (pedroagostinhoaptoide) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/WantMoreViewFooter.kt]: Fixed

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/MoreBonusViewHeader.kt (hunk 2 above)
Comment (pedroagostinhoaptoide) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/MoreBonusViewHeader.kt]: Fixed

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/MoreBonusViewFooter.kt (hunk 1 above)
Comment (pedroagostinhoaptoide) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/backgrounds/MoreBonusViewFooter.kt]: Fixed

**Review by GuiFPires — APPROVED** [review]

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Forward.kt [hunk 6]
```diff
@@ -5,44 +5,74 @@ import androidx.compose.foundation.layout.size
...
 }
 
-fun getForward(iconColor: Color): ImageVector = ImageVector.Builder(
-  name = "Forward",
-  defaultWidth = 24.dp,
-  defaultHeight = 24.dp,
-  viewportWidth = 24f,
-  viewportHeight = 24f,
+fun getForward(
+  backgroundColor: Color = Palette.Primary,
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Forward.kt:31]: Let's follow the convention we have about colors by naming them as color1, color2 etc.

**Review by G0retZ — CHANGES_REQUESTED** [review]

File: app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Forward.kt (hunk 6 above)
Comment (pedroagostinhoaptoide) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/drawables/icons/Forward.kt]: Fixed

**Review by G0retZ — APPROVED** [review]

**Pull request merged**